ui/src/message_popup/experience_page.jsx

```diff
@@ -168,25 +157,27 @@ export default React.createClass({
   },
 
   renderInstructions() {
+    const {scaffolding, gameSession} = this.state;
     return (
-      <InstructionsCard 
-        sessionLength={this.state.sessionLength}
-        onStartPressed={this.onStartPressed}
-        email={this.state.email}
-        itemsToShow={this.props.query}
-        helpType={this.state.helpType}
-        />);
+      <VelocityTransitionGroup enter={{animation: "callout.pulse", duration: 500}} leave={{animation: "slideUp"}} runOnMount={true}>
+        <div>
+          <InstructionsCard 
+           itemsToShow={this.props.query}
+           />
+          <ScaffoldingCard
+            email={gameSession.email}
+            scaffolding={scaffolding}
+            itemsToShow={this.props.query}
+            saveScaffoldingAndSession={this.onSaveScaffoldingAndSession}
```

Comment:
Let's call something that describes what's happening conceptually, more than the pieces involved.  How about `onSessionStarted` or `onSessionConfigured`?  It's generally helpful to use the `on` prefix to indicate that something is a callback function, and this clarifies that this is only getting called at the end (not when the scaffolding values change).